### ISSUE TITLE
About changing the tab in tabbar

### ISSUE THREAD

**Issue by Nautiyalsachin** [stoncle/ZOWVideoPlayer#1]
I am using the tab bar for making the Instagram-like app. But whenever I am changing the tabs I am unable to get the video player on the view will appear, its always returning me nil. Don't know why but for every other case it's working superb only for this case, have you included any observer for that, let me know if there is any.

**Comment by stoncle** [comment]
@Nautiyalsachin NO, there are no observer for that. By your case, I made a test project which contains a UITabbarController but I don't get the same error, the video player is not nil when viewWillAppear get called.
Can you describe your using case more detail? By the way, I will expose more playing status property outside for users to make control more easily so you don't need to access the video player directly.

**Comment by Nautiyalsachin** [comment]
I solved that issue it was from my side, but it will be good if there are other playing status.One more thing, I am using it in the table view and sometimes the video player just show me a white color video view and after some time it will load the video again and play it. Don't know why it's happening but yeah it's there (mostly happening when table view cell are getting reload) . And I am enjoying the ZOWVideoPlayer it's very cool. Thanks.

**Comment by stoncle** [comment]
@Nautiyalsachin Your problem on tableview is because when you call play video method on cell reload and video will be play from begin again. This will be solved after I expose the play status outside, it is easy to be solved, and no need to change your current code.

**Comment by Nautiyalsachin** [comment]
@stoncle Great. Waiting for the update then or you can tell me how to get the current stauts.

**Comment by Nautiyalsachin** [comment]
Sometimes the video is not loading when I am reloading the table view, its just showing me the background color of my cell. And yeah sometime it shows the instagram animation for downloading the video when its already there.

**Comment by stoncle** [comment]
@Nautiyalsachin yeah,same problem with above. thank you for your reply and I will fix it as soon as possible :-)
